lib/inspec/dependencies/requirement.rb

```diff
@@ -10,7 +10,18 @@ module Inspec
   class Requirement
     def self.from_metadata(dep, cache, opts)
       raise 'Cannot load empty dependency.' if dep.nil? || dep.empty?
-      new(dep[:name], dep[:version], cache, opts[:cwd], opts.merge(dep))
+
+      req_path = opts[:cwd]
+
+      if dep[:path]
+        req_path = File.join(req_path, dep[:path])
```

Comment:
I think this will only work if the `path` in the requirement is an absolute path. What if I have an absolute path in my metadata?

```yaml
depends:
  - name: foo
    path: '/path/to/foo'
```

If my cwd is `/home/aleff/profiles`, this would set the `req_path` to `/home/aleff/profiles/path/to/foo` which won't exist... correct?

You probably want to look into using `File.expand_path` here instead which handles both relative and absolute path expansion.

```
[1] pry(main)> Dir.pwd
=> "/Users/aleff/profiles/test1"
[2] pry(main)> File.expand_path('../test2', Dir.pwd)
=> "/Users/aleff/profiles/test2"
[3] pry(main)> File.expand_path('/opt/chef/embedded', Dir.pwd)
=> "/opt/chef/embedded"
```